### ISSUE TITLE
Investigate why code for database coverage appears to give an incorrect total

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#720]
Devon for 1881 on database coverage shows 8990 records loaded. It shows that there are 3 pieces on line but only gives the total for the first 2.
Using the Manage Pieces for Devon shows 3 pieces with records loaded for a total of 12380.

**Comment by Captainkirkdawson** [comment]
The total number of individual records  is 37,365,159 according to mongodb. That is the total shown by database coverage for brazza, drill, howler and saki. Colobus reports 37,148,266

**Comment by Vino-S** [comment]
Yes. Looking at freecen coverage page code. However if we have a bug in the code then it should affect brazza, drill, howler and saki too.

**Comment by Captainkirkdawson** [comment]
@Vino-S I agree 100% that almost all bugs would affect all servers not just one. There are at least 2 situations I can think of that could lead to a difference. 1) the code is using values from a file rather than just the database 2) it is using a leftover value "left over from a previous calculation instead of clearing or resetting it.

**Comment by Captainkirkdawson** [comment]
@FreecenBren I think you can leave it up to @Vino-S and myself to dig into the matter of why one server is giving an incorrect total. What I think you need to focus on if the missing records between FC1 and FC2 which is as you say 59.902 records. Could I suggest that we focus comments on that in story #256

**Comment by FreecenBren** [comment]
Good idea. Will do.

**Comment by Captainkirkdawson** [comment]
the answer and solution lies in the controller

    # Since get_index_stats is slow (several seconds on production), we cache
    # the result so subsequent hits will be faster. Be sure to do
    # Rails.cache.delete when updates are done (when records or parms are
    # added/deleted/updated in db)
    @all_pieces = Rails.cache.fetch("freecen_coverage_index", :expires_in => 7.days) do

Actually creates a database coverage cache that lasts  for 7 days.

We need to add Rails.cache.delete to the update_freecen rake task

**Comment by Captainkirkdawson** [comment]
Thinking about the process more adding it to the update_freecen rake task would be ineffective. The 
Rails.cache.delete has to be run on each app server and the update runs on only one.
Perhaps we should determine if we actually need to cache the results given the change in servers since it was added.

**Comment by Captainkirkdawson** [comment]
In meantime. Add step to delete cache in FreeCen update process document
